agbenchmark/benchmarks.py

```diff
@@ -1,17 +1,36 @@
-# sourcery skip: avoid-global-variables
+import asyncio
 import sys
 
-from dotenv import load_dotenv
+from agent_protocol_client import Configuration
 
+import agbenchmark.e2b_boilerplate
 
-def run(task: str) -> None:
+configuration = Configuration(host="http://localhost:8915")
+
+
+async def run_specific_agent(task: str) -> None:
```

Comment:
you can start this directly, no need to run it in subprocess. Check this https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks/pull/278 and this: https://github.com/e2b-dev/smol-developer/tree/benchmarks/agbenchmark

The integration should be simple as that, but we will need new version of benchmarks

Reply:
Thanks, I'll take a look at your PR on AGBenchmark see if I can get that merged asap

Reply:
Thanks 🙏🏻